src/ApiRequestPanel.js

```diff
@@ -419,6 +430,19 @@ export class ApiRequestPanel extends EventsTargetMixin(HeadersParserMixin(LitEle
     this._authPopupLocation = value;
     this._updateRedirectUri(value);
   }
+
+  get selectedServer() {
+    return this.baseUri || this._selectedServer || '';
+  }
+
+  set selectedServer(value) {
+    const old = this._selectedServer;
+    if (old === value) {
+      return;
+    }
+    this._selectedServer = value;
```

Comment:
You need to add `this.requestUpdate('selectedServer', value);` otherwise the element won't know that the UI should be re-rendered.

Reply:
`_selectedServer` is declared in the properties, so when we set `selectedServer`, it changes `_selectedServer` and re-renders